prow/plugins/help/help.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package help
+
+import (
+	"regexp"
+
+	"github.com/Sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "help"
+
+var (
+	helpLabel = "help-wanted"
+	helpRe    = regexp.MustCompile(`(?mi)^/help\s*$`)
+)
+
+type assignEvent struct {
+	action    string
+	body      string
+	login     string
+	org       string
+	repo      string
+	url       string
+	number    int
+	issue     github.Issue
+	assignees []github.User
+	hasLabel  func(label string) (bool, error)
+}
+
+func init() {
+	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment)
+	plugins.RegisterIssueHandler(pluginName, handleIssue)
+}
+
+type githubClient interface {
+	CreateComment(owner, repo string, number int, comment string) error
+	AddLabel(owner, repo string, number int, label string) error
+	RemoveLabel(owner, repo string, number int, label string) error
+}
+
+func handleIssueComment(pc plugins.PluginClient, ic github.IssueCommentEvent) error {
+	// Only consider open issues.
+	if ic.Issue.IsPullRequest() || ic.Issue.State != "open" {
+		return nil
+	}
+
+	ae := assignEvent{
+		action:    ic.Action,
+		body:      ic.Comment.Body,
+		login:     ic.Comment.User.Login,
+		org:       ic.Repo.Owner.Login,
+		repo:      ic.Repo.Name,
+		url:       ic.Comment.HTMLURL,
+		number:    ic.Issue.Number,
+		issue:     ic.Issue,
+		assignees: ic.Issue.Assignees,
+		hasLabel:  func(label string) (bool, error) { return ic.Issue.HasLabel(label), nil },
+	}
+	return handle(pc.GitHubClient, pc.Logger, ae)
+}
+
+func handleIssue(pc plugins.PluginClient, i github.IssueEvent) error {
```

Comment:
I'm not entirely sure, tbh. I know prow doesn't make guarantees about the order it runs commands/plugins. Likely a race condition? @spxtr ?